packages/video_player/video_player_android/android/build.gradle

```diff
@@ -48,11 +48,12 @@ android {
     }
 
     dependencies {
-        def exoplayer_version = "2.18.7"
-        implementation "com.google.android.exoplayer:exoplayer-core:${exoplayer_version}"
-        implementation "com.google.android.exoplayer:exoplayer-hls:${exoplayer_version}"
-        implementation "com.google.android.exoplayer:exoplayer-dash:${exoplayer_version}"
-        implementation "com.google.android.exoplayer:exoplayer-smoothstreaming:${exoplayer_version}"
+        def exoplayer_version = "1.1.1"
```

Comment:
version 1.1.1 corresponds to exoplayer 2.19.1
version 1.3.1 is the latest one https://github.com/androidx/media/blob/release/RELEASENOTES.md

Reply:
Can you bump to the latest version?

Reply:
[These are the steps](https://github.com/flutter/packages/blob/9a2490419268365911191f9d8388f986111bcc24/.ci/targets/android_build_all_packages.yaml#L14-L25) to run that locally.

Reply:
reproduced, thanks (one also needs `running prepare_tool.sh` and `create_all_packages_app.sh legacy`)

the issue could be simplified to:
- use gradle 6.x
- bump guava version to 32.1.x (media3-1.3.1 uses guava 32.1.3-android) in any plugin (e.g. video_player_android like here or camera_android_camerax)

then we'd face following - https://github.com/google/guava/issues/6801
gradle 7+ handles this fine with 32.1.2+
gradle 6 seem to need workarounds described at https://github.com/google/guava/releases/tag/v32.1.0 (not sure, investigating)

so, this is a generic issue with legacy gradle 6 project and guava 32, should I file it separately and use compatible media3 version in this pr?

Reply:
added workaround for gradle 6.x